downstream/modules/platform/proc-new-aap-instance-upgrade.adoc

```diff
@@ -25,6 +25,6 @@ NOTE: Make note of all instance groups from your original Tower instance. You wi
 
 . Log in to your new instance of Tower.
 . On the side panel, navigate to menu:Administration[Instance groups].
```

Comment:
Using directional information such as side...is a violation, suggest changing to:

In the navigation pane, select menu:menu:Administration[Instance groups].